django/core/checks/security/csp.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+
+from .. import Tags, Warning, register
+
+W026 = Warning(
+    "Each enabled CSP setting (SECURE_CSP, SECURE_CSP_REPORT_ONLY) must be a "
+    "dictionary.",
+    id="security.W026",
+)
+
+
+@register(Tags.security)
+def check_csp_directives(app_configs, **kwargs):
```

Comment:
```suggestion
def check_csp_settings(app_configs, **kwargs):
```

I think, since we dropped `DIRECTIVES`